.github/workflows/locize-push-missing-keys.yml

```
@@ -0,0 +1,26 @@
+      - name: Set Up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Install locize CLI
+        run: npm install -g locize-cli
+
+      - name: Push Missing Translation Keys to locize
+        run: |
+          cd client/src/locales
+          locize save-missing --api-key ${{ secrets.LOCIZE_API_KEY }} --project-id ${{ secrets.LOCIZE_PROJECT_ID }} --language en
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)